cluster-operator/src/test/java/io/strimzi/operator/cluster/model/EntityOperatorTest.java

```diff
@@ -50,10 +50,20 @@
 public class EntityOperatorTest {
 
     private static final KafkaVersion.Lookup VERSIONS = new KafkaVersion.Lookup(new StringReader(
-            "2.0.0 default 2.0 2.0 1234567890abcdef 2.0.x\n" +
-            "2.1.0         2.1 2.0 1234567890abcdef 2.1.x"),
-            map("2.0.0", "strimzi/kafka:latest-kafka-2.0.0",
-                    "2.1.0", "strimzi/kafka:latest-kafka-2.1.0"), emptyMap(), emptyMap(), emptyMap()) { };
+            "- version: 2.2.1\n" +
+                    "  format: 2.2\n" +
+                    "  protocol: 2.2\n" +
+                    "  checksum: ABCDE1234\n" +
+                    "  third-party-libs: 2.2.x\n" +
+                    "  default: false\n" +
+                    "- version: 2.3.0\n" +
+                    "  format: 2.3\n" +
+                    "  protocol: 2.3\n" +
+                    "  checksum: ABCDE1234\n" +
+                    "  third-party-libs: 2.3.x\n" +
+                    "  default: true"),
+            map("2.2.1", "strimzi/kafka:latest-kafka-2.2.1",
+                    "2.3.0", "strimzi/kafka:latest-kafka-2.3.0"), emptyMap(), emptyMap(), emptyMap()) { };
```

Comment:
You are right about this, refactoring this was a horror show but I got so far into I thought I better finish it. So should I add something to the appropriate Utils class?

Reply:
Yeah, something in TestUtils. Or a new `KafkaVersionTestUtil`

Reply:
Ok I added `KafkaVersionTestUtils` class that is the single source of all version and image strings and refactored all the tests to use this new class. I think I refactored everything that used a version string. The hardest class was the `cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaUpdateTest.java`, I changed it to use `PREVIOUS` and `LATEST` rather than specific versions. Would be worth checking that I have got the logic correct in those tests (I am pretty sure it still tests what was intended).